### ISSUE TITLE
[GH Permissions] Need permission to push a branch 

### ISSUE THREAD

**Issue by dgolubovicTT** [tenstorrent/tt-metal#13006]
## What you are trying to do...

Push a branch

## Who needs permissions...

I need the permission for myself @dgolubovic

**Comment by tt-rkim** [comment]
I don't believe we should be adding personal users. Can you please clarify why you need both? Could you just copy the branch diff to a machine which has your SSH key for your TT user?

**Comment by dgolubovicTT** [comment]
Sorry for the misunderstanding, but I don't need the permission for personal use, I need it for @dgolubovicTT account. I am trying to push my branch to remote repository using usual ssh key but I see this error:

<img width="807" alt="image" src="https://github.com/user-attachments/assets/9ffa2f10-e196-4b03-98eb-7f29313e1be7">

**Comment by TT-billteng** [comment]
is this resolved?
